SeleniumTestXUnit/Tests/Features/Project/Post.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Project Creation
+    As a logged in user, the user should be able to create a project.
+    
+    Background:
+        Given the user is logged in
+
+    @Smoke @Regression @go.to.login.page @delete.project
+    Scenario: Create a new project
+        When the user clicks the New Project button
+            And inputs a new project name
+            And clicks the Add button
+        Then a new project with the chosen name should be displayed in the projects list
+
+    @Negative @go.to.login.page
+    Scenario Outline: Create a new project with an empty name
+        When the user clicks the New Project button
+            And inputs an empty string as the project name
+            And clicks the Add button
```

Comment:
The Add button in the UI is shown as "Add New Project", please try to describe the UI as exactly as possible on the scenarios.

Reply:
Actually here in `And clicks the Add button` I'm refencing the the little button that appears after you click on the "Add New project" button.